pallets/gear-eth-bridge/src/tests.rs

```diff
@@ -227,6 +251,10 @@ fn bridge_queue_root_changes() {
     new_test_ext().execute_with(|| {
         run_to_block(WHEN_INITIALIZED);
 
+        assert_ok!(GearEthBridge::set_fee(
```

Comment:
remove these from everywhere

impl test that tests set_fee extrinsic directly:
- value before
- set
- value after 
- change 
- value after 

expand 1 test for extrinsic:
- user is charged for transport fee and value is moved to destination

expand 1 test for builtin:
- builtin fails with error on insufficient value applied

Reply:
Fixed